web/modules/custom/vote_utility/translations/es.po

```diff
@@ -92,7 +133,7 @@ msgid "Voter registration deadlines for the @election_date election. @link"
 msgstr "Plazo de inscripción de votantes para las elecciones del @election_date"
 
 msgid "Find state and local election dates."
-msgstr ""
+msgstr "Encuentre las fechas de las elecciones locales y estatales."
```

Comment:
@clmedders This is not showing up on the state pages at all in Spanish, but English shows. There might be a bug the translated link.

Reply:
i made a fix in the tempalte to address this thanks for pointing that out!

Reply:
I'm just going to add the (in English) for it too, other than that, link is working now